core/network/fabric-interop-cc/interfaces/asset-mgmt/asset_locks.go

```diff
@@ -0,0 +1,828 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package main
+
+import (
+    "fmt"
+    "errors"
+    "time"
+    "strconv"
+    "encoding/json"
+
+    "github.com/golang/protobuf/proto"
+    "github.com/hyperledger/fabric-chaincode-go/shim"
+    log "github.com/sirupsen/logrus"
+    "github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/interfaces/asset-mgmt/protos-go/common"
+)
+
+
+const(
+    defaultTimeLockMillis = 5 * 60 * 1000      // 5 minutes
+)
+
+
+// AssetManagement provides functions to manage asset locks by calling suitable functions in the interop contract
+type AssetManagement struct {
+    shim.Chaincode
+    interopChaincodeId string
+}
+
+
+// Utility functions
+func (am *AssetManagement) Configure(interopChaincodeId string) {
+    am.interopChaincodeId = interopChaincodeId
+}
+
+
+// Ledger transaction (invocation) functions
+
+func (am *AssetManagement) AddFungibleAssetCount(stub shim.ChaincodeStubInterface, assetType string, numUnits int) (bool, error) {
+    var errorMsg string
+
+    if len(am.interopChaincodeId) == 0 {
+        errorMsg = "Interoperation chaincode ID not set. Run the 'Configure(...) function first."
```

Comment:
looks like the closing single quote is missing after `Configure()` ... comment applies to all other places where this message is duplicated in other functions

Reply:
Right. I'll fix it.

Reply:
Done